src/org/labkey/test/AssayAPITest.java

```diff
@@ -550,4 +504,30 @@ public void testGpatImportRunWithPlate() throws Exception
 
         verifyPlateMetadata(resp, expectedRows);
     }
+
+    public void createPlateTemplate()
```

Comment:
Did you mean to make this public and leave it in this class? It looks like it is in the APIAssayHelper class too.

Reply:
As per our discussion each test class is going to have its own createTemplateTest